Dockerfile

```diff
@@ -31,22 +31,25 @@ RUN curl -Lo /tmp/envconsul.zip https://releases.hashicorp.com/envconsul/0.9.0/e
     rm /tmp/envconsul.zip
 
 
-
 COPY Gemfile Gemfile.lock /app/
 COPY package.json yarn.lock /app/
 RUN gem install bundler
+RUN yarn
 
 RUN useradd -u 10000 app -d /app
 RUN chown -R app /app
 USER app
+ENV BUNDLE_PATH=vendor/bundle
 
+# RUN --mount=type=cache,id=gem,sharing=locked,uid=10000,target=/app/vendor bundle package 
```

Comment:
@whereismyjetpack can you remove any extraneous comments? It looks like these were part of some testing process.